backend/src/main/java/quarano/department/web/TrackedCaseController.java

```diff
@@ -186,10 +191,13 @@
 	// PUT Mapping for transformation into index case
 
 	@PutMapping("/hd/cases/{identifier}")
-	HttpEntity<?> putCase(@PathVariable TrackedCaseIdentifier identifier,
-			@RequestBody TrackedCaseDto.Input payload,
+	HttpEntity<?> putCase(@PathVariable TrackedCaseIdentifier identifier, @RequestBody TrackedCaseDto.Input payload,
```

Comment:
What line length did you set?

Reply:
Probably you don't have a wrong setting. The code rather had a wrong line break before your change, which you have corrected. I will commit to this among other things.